pnpm-workspace.yaml

```diff
@@ -14,4 +14,5 @@ overrides:
   # A later wrangler minor dropeed support for node 18
   'wrangler': '4.14.1'
 # Wait until three days after release to install new versions of packages
-minimumReleaseAge: 4320
+# TODO: uncomment before merging into main
+# minimumReleaseAge: 4320
```

Comment:
This was preventing pnpm from using the preview release so I disabled it. I added a task on the Astro 6 project to remember to re-enable it later